Disable checks against macros that do not exist in ffmpeg-2.x

Patch by aballier.

--- a/media/ffmpeg/ffmpeg_common.cc
+++ b/media/ffmpeg/ffmpeg_common.cc
@@ -786,7 +786,9 @@ TEST_PRIMARY(SMPTE170M);
 TEST_PRIMARY(SMPTE240M);
 TEST_PRIMARY(FILM);
 TEST_PRIMARY(BT2020);
+#if LIBAVUTIL_VERSION_INT > AV_VERSION_INT(55,5,0)
 TEST_PRIMARY(SMPTEST428_1);
+#endif
 
 TEST_TRANSFER(RESERVED0);
 TEST_TRANSFER(BT709);
@@ -804,8 +806,10 @@ TEST_TRANSFER(BT1361_ECG);
 TEST_TRANSFER(IEC61966_2_1);
 TEST_TRANSFER(BT2020_10);
 TEST_TRANSFER(BT2020_12);
+#if LIBAVUTIL_VERSION_INT > AV_VERSION_INT(55,5,0)
 TEST_TRANSFER(SMPTEST2084);
 TEST_TRANSFER(SMPTEST428_1);
+#endif
 
 TEST_COLORSPACE(RGB);
 TEST_COLORSPACE(BT709);
